lib/oban.ex

```diff
@@ -647,6 +647,43 @@ defmodule Oban do
     |> Query.retry_job(job_id)
   end
 
+  @doc """
+  Retries all discarded and retryable jobs. For more complex mass retries, see `retry_all_jobs/2`.
+
+  ## Example
+
+      Oban.retry_all_jobs()
+      :ok
+  """
+  @doc since: "2.8.0"
+  @spec retry_all_jobs :: :ok
+  def retry_all_jobs() do
```

Comment:
This clause can be removed. With some defaults, both clauses are covered by `retry_all_jobs/2`.